boards/frdm-kw41z/Makefile.dep

```diff
@@ -0,0 +1,7 @@
+USEMODULE += devfs
+# USEMODULE += mtd_spi_nor
+
+ifneq (,$(filter saul_default,$(USEMODULE)))
+#   USEMODULE += saul_gpio
```

Comment:
Well the thing is that SAUL seems to "work" or at least active on this board, but giving weird numbers. Is it possible to disable SAUL on this board until we have the configuration files?